backend/migrations/versions/6c9a8afee4e4__alembic_sync.py

```diff
@@ -0,0 +1,163 @@
+"""_describe_changes_shortly
+
+Revision ID: 6c9a8afee4e4
+Revises: f6cd4ba7dd8d
+Create Date: 2024-02-13 11:09:39.387899
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '6c9a8afee4e4'
+down_revision = 'f6cd4ba7dd8d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('redshiftcluster_datasettable')
+    op.drop_table('dataset_quality_rule')
+    op.drop_table('tag')
+    op.drop_table('redshiftcluster_dataset')
+    op.drop_table('item_tags')
+    op.drop_table('dataset_table_profiling_job')
+    op.drop_table('redshiftcluster')
+    op.drop_table('group_member')
```

Comment:
Aren't line 29 and line 30 doing the same?

Reply:
If you use alembic lib to create a foreign key in model, It will result create_foreign_key. Constraint is more general think. I would leave it this way